### ISSUE TITLE
Shorten surface temperature responsive description

### ISSUE THREAD

**Issue by arouinfar** [phetsims/greenhouse-effect#132]
For #130 

Currently, this phrase repeats every 2 seconds so long as the temperature has increased by at least 0.1 K:

> Surface temperature {{warming/cooling}}, now {{255.2 Kelvin}}.

This phrase quickly becomes very repetitive with a screen reader, so we propose these changes:

* - [x] Increase the spacing between the statements to every 4 seconds.
* - [x] When the `concentrationProperty` changes, state the full phrase. 
  * Surface temperature {{cooling}}, now {{266.1 Kelvin}}.
* - [x] Otherwise, alternate between these two abbreviated options
  * {{Warming/Cooling}}.
  * {{Warming/Cooling}}, now {{266.1 Kelvin}}. (Every 3rd time.)
  
Here's an example of what happens after the user decreases the GHG concentration (ignoring all other descriptions):
  1.  Surface temperature {{cooling}}, now {{266.1 Kelvin}}.
  2. {{Cooling}}.
  3. {{Cooling}}.
  4. {{Cooling}}, now {{261.3 Kelvin}}.
  5. Phrases 2-4 repeat until equilibrium statement.

**Comment by jessegreenberg** [comment]
Sounds good. I am assuming that we just forever hear "Cooling" or "Warming" if the surface thermometer is invisible.

**Comment by arouinfar** [comment]
> I am assuming that we just forever hear "Cooling" or "Warming" if the surface thermometer is invisible.

Correct! Similarly, the first response should be "Surface temperature {{warming/cooling}}."

**Comment by jessegreenberg** [comment]
OK these changes are done. I made one other adjustment - In the current implementation nothing is spoken immediately after the concentration changes, it just waits until the next interval to speak content. So increasing the spacing to 4 seconds meant that you would change the concentration and then ~4 seconds later hear 
>"Surface temperature {{cooling}}, now {{266.1 Kelvin}}."

I made it so that whenever you change the concentration you hear alerts within ~2 seconds, otherwise it is on a four second interval.

@arouinfar and @Matthew-Moore240 over to you to review changes.
